docs/concepts/services-networking/service.md

```diff
@@ -585,6 +585,84 @@ annotation:
 Since version 1.3.0 the use of this annotation applies to all ports proxied by the ELB
 and cannot be configured otherwise.
 
+#### ELB Access Logs on AWS
+
+There are several annotations to manage access logs for ELB services on AWS.
+
+The annotation `service.beta.kubernetes.io/aws-load-balancer-access-log-enabled`
+controls whether access logs are enabled.
+
+TODO: Describe each annotation here?
+
+```yaml
+    metadata:
```

Comment:
All these examples are indented by 4 spaces because all the other examples in this file are. Should I un-indent them all? It seems a little weird that they're indented, as metadata is not indented in a full service yaml.

```yaml
apiVersion: v1
kind: Service
metadata:
  name: my-service
  annotations:
    service.beta.kubernetes.io/aws-load-balancer-type: "nlb"
  namespace: default
spec:
  externalTrafficPolicy: Cluster
  ports:
  - name: http
    port: 80
    protocol: TCP
    targetPort: 80
  selector:
    app: nginx
  type: LoadBalancer
```

Reply:
@micahhausler I leave that to your judgment. The readability seems better to me with indentations. (I fed it through a linter and got the same results.)